requirements/requirements.txt

```diff
@@ -1,6 +1,6 @@
 numpy>=1.18.2
 onnx>=1.7.0
-pytorch-lightning>=1.0.3
+pytorch-lightning==1.0.4
```

Comment:
why strict equality here?

Reply:
PTL 1.0.5 was causing an exp_manager unit test to fail.

@titu1994 has already created a patch: https://github.com/NVIDIA/NeMo/pull/1407

I can revert it or change it, but I fixed it (at least temporarily) so I could unblock the CI.